### ISSUE TITLE
Small semantic issue in no-unneeded-ternary documentation example

### ISSUE THREAD

**Issue by marcelozarate** [eslint/eslint#4937]
Inside the documentation file:
[no-unneeded-ternary](https://github.com/eslint/eslint/blob/master/docs/rules/no-unneeded-ternary.md)

The first example shows:
```javascript
// Bad
var isYes = answer === 1 ? true : false;

// Good
var isYes = answer === 1;


// Bad
var isNo = answer === 1 ? false : true;

// Good
var isYes = answer !== 1;
```

Shouldn't the last line be 
```javascript
var isNo = answer !== 1; 
```
instead?

**Comment by eslintbot** [comment]
@marcelozarate Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by btmills** [comment]
Yep, looks like it. @marcelozarate want to submit a PR? ([Contributing guide, for your reference.](http://eslint.org/docs/developer-guide/contributing/))

**Comment by marcelozarate** [comment]
Ok.
